doc/releases/changelog-dev.md

```diff
@@ -10,6 +10,9 @@
   function fails because the Qiskit converter is missing.
   [(#5218)](https://github.com/PennyLaneAI/pennylane/pull/5218)
 
+* Qiskit operators can be converted into PennyLane operators using `qml.from_qiskit_op`.
```

Comment:
For clarity, could we say "A Qiskit `SparsePauliOp` can be converted" instead of "Qiskit operators"?

Reply:
Sure! I opted for "Qiskit operators" to better match the function name but I'm OK with being more specific too.